### PR TITLE
Docker Austin meetup

### PR DESCRIPTION
**Pull request by everett-toews**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-cd8a18d815/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-61d9410332/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-8c372a298c/)

File: _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: "Docker Austin: How Do I Even Swarm?"
+date: 2016-04-11
+comments: true
+author: Everett Toews <everett.toews@rackspace.com>
+published: true
+excerpt: >
+  Docker Austin is a meetup that runs out of one of the Austin offices of Rackspace on the first Thursday of every month. Last Thursday I gave a presentation there titled How Do I Even Swarm?
```
Comment (kmsholcomb) [inline _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md]: "How Do I Even Swarm?"

(Enclose the title of the presentation in quotation marks.)

File: _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+categories:
+  - Docker
+  - Swarm
+  - Carina
+  - Rackspace
+  - News
+authorIsRacker: true
+---
+
+{{ page.excerpt }} I've since changed the title to Effective Docker Swarm.
```
Comment (kmsholcomb) [inline _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md]: "Effective Docker Swarm."

File: _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+This is a meetup I've been attending for the past year and know everyone pretty well already. If you're in the Austin area, you're always welcome to join us at [Docker Austin](http://www.meetup.com/Docker-Austin/). Thursday night was an audience of 40-50 and they were highly engaged to say the least. I got a lot of great, detailed questions on Carina and Swarm during the talk. I literally spent twelve minutes on the Carina architecture slide while getting peppered with questions.
```
Comment (kmsholcomb) [inline _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md]: I've been attending this meetup for the past year...

...40-50, and they were highly engaged, to say ...

... 12 minutes ...

File: _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+Here's the video.
+
+<iframe width="800" height="450" src="https://www.youtube.com/embed/SShmLUlrals?rel=0" frameborder="0" allowfullscreen></iframe>
+
+The [slide deck](https://everett-toews.github.io/effective-docker-swarm/slides/index.html) contains a fair amount of code and commands that you can follow. Click on the presentation below, and use the left/right keys or swipe left/right to navigate.
```
Comment (kmsholcomb) [inline _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md]: ... Click the following presentation, and ...

File: _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+<iframe width="800" height="450" frameborder="0" src="https://everett-toews.github.io/effective-docker-swarm/slides/index.html">
+  Effective Docker Swarm slides
+</iframe>
+
+I also got a lot of feedback on the presentation itself at the end, which was very helpful. I'm going to rework it a bit.
+
+I'm going to put more emphasis on:
```
Comment (kmsholcomb) [inline _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md]: ... I'm going to rework it a bit, putting more emphasis on the following topics:

(Merge the two paragraphs.)

File: _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+* Delineation of Carina and Swarm
+* Benefits of Carina
+* The competitive landscape (Swarm vs Kubernetes vs Mesos)
```
Comment (kmsholcomb) [inline _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md]: vs.

File: _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+I'm going to put less emphasis on:
+
+* Carina implementation details
+ * An interesting subject but too easy to get bogged down
+ * Takes emphasis away from developing and deploying apps on Carina
+ * Better to say "Let's chat about it after the presentation"
```
Comment (kmsholcomb) [inline _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md]: I'm going to put less emphasis on Carina implementation details, for the following reasons:

- An interesting subject, but too easy to get bogged down
- Takes emphasis away from developing and deploying apps on Carina
- Better to say "Let's chat about it after the presentation"

File: _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+Overall it was a great meetup and I really learned a lot from the audience.
```
Comment (kmsholcomb) [inline _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md]: ... a great meetup, and I ...

File: _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+If you're in Austin on April 26 (perhaps for the OpenStack Summit), join the Carina team at the Rackspace Cantina using this [free invite](http://www.cvent.com/d/pfq4wl/4W). You can hear from myself on Effective Docker Swarm, Ross Kukulinski of Yodlr on Using Carina in Production, and Tim Hockin of Google on Pain Management for Containerized Apps. Grab a t-shirt, have a drink and a bite to eat, and chat containers with us.
```
Comment (kmsholcomb) [inline _posts/2016-04-11-docker-austin-how-do-i-even-swarm.md]: ... free invitation ...

**Comment by kmsholcomb** [comment]
Hey @everett-toews: I left just a few suggested edits, in comments in the file. Please let me know if you have any questions. Thanks!

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-d05614edb2/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-b1a4113f83/)

**Pull request merged**